portfolio/src/main/java/com/google/sps/servlets/MyOwnServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles request sent to /funny */
+@WebServlet("/funny")
+public class MyOwnServlet extends HttpServlet {
```

Comment:
Doesn't really matter too much but `FunnyServlet` would be a more accurate name. :)

Reply:
Yeah, you're right.